src/newViews/Tagging/Sources/BulkUpload/Upload/index.tsx

```diff
@@ -14,12 +26,32 @@ function Upload(props: Props) {
         className,
     } = props;
 
+    const handleChange = (response: google.picker.ResponseObject) => {
+        console.warn('values', response.docs);
```

Comment:
Do we need this here? Is it to be handled in the next issue? Just asking coz your PR says that there are no console.warn for debugging.